lib/modules/modules.go

```diff
@@ -213,6 +231,9 @@ const (
 	ProductTypeEUB ProductType = 2
 )
 
+// SupportType is the type of support.
+type SupportType proto.SupportType
```

Comment:
Ditch the type entirely? Or does it serve some purpose I'm missing?

Reply:
Not really, I was following `ProductType`, which is duplicated in this file and in the proto file. But I do prefer ditching the type.